daemon/exec.go

```diff
@@ -293,15 +296,14 @@ func (daemon *Daemon) ContainerExecStart(ctx context.Context, name string, optio
 	select {
 	case <-ctx.Done():
 		logrus.Debugf("Sending TERM signal to process %v in container %v", name, c.ID)
-		daemon.containerd.SignalProcess(ctx, c.ID, name, signal.SignalMap["TERM"])
+		daemon.containerd.SignalProcess(context.Background(), c.ID, name, signal.SignalMap["TERM"])
 
 		timeout := time.NewTimer(termProcessTimeout)
```

Comment:
Process has 10s to process the SIGTERM - in the test case SIGTERM is just ignored, so this will have to wait 10 seconds.

Reply:
Oh, so obvious I didn't saw it... thanks